### ISSUE TITLE
Ra not rendering correctly tagalog 

### ISSUE THREAD

**Issue by p0txky** [googlefonts/noto-fonts#2178]
### Title 
   Ra not rendering correctly tagalog 
### Font 
   'NotoSansTagalog-Regular.ttf'.

### Where the font came from, and when
   google-noto-sans-tagalog-fonts 

### Font Version
  Version 2.000

### OS name and version
  Fedora 35

### Application name and version
  Web app like youtube

### Character data
  U+1707

### Screenshot
![image](https://user-images.githubusercontent.com/65337802/144792409-87b933fe-942b-4537-ab57-333c0763a437.png)

**Comment by Mercury13** [comment]
A new character of U14, they just did not implement it.
Here’s my quick fix, probably not completely functional.
[NotoSansTagalog-Regular.zip](https://github.com/googlefonts/noto-fonts/files/7667817/NotoSansTagalog-Regular.zip)

**Comment by Mercury13** [comment]
@p0txky 
I returned home and saw that one of new characters has no anchors and the other is improperly hinted.
Here’s a new version.
https://github.com/Mercury13/unicodia/blob/main/Fonts/NotoSansTagalog-Regular.ttf

**Comment by p0txky** [comment]
@marekjez86 How to contribute? I only extract the tff given by @Mercury13 to ufo. Installing the font seems to be working.

**Comment by tommai4881** [comment]
Friedrich Brennan (ctrlcctrlv) has an amazing Tagalog unicode font which
may be the update of Noto Sans Tagalog. It also has Tagalog letter Ra
(u+170d)

Vào Th 4, 15 thg 12, 2021 vào lúc 11:08 p0txky ***@***.***>
đã viết:

**Comment by p0txky** [comment]
@tommai4881 yep he opened an issue long time ago but no update its close now. Need fix to all os or atleast I'm using linux.
